src/containers/MyNdla/components/UserInfo.tsx

```diff
@@ -7,60 +7,77 @@
  */
 
 import { useTranslation } from "react-i18next";
-import styled from "@emotion/styled";
-import { misc, spacing } from "@ndla/core";
-import { Text } from "@ndla/typography";
+import { Text } from "@ndla/primitives";
+import { styled } from "@ndla/styled-system/jsx";
 import { GQLMyNdlaPersonalDataFragmentFragment } from "../../../graphqlTypes";
 
 interface Props {
   user: GQLMyNdlaPersonalDataFragmentFragment | undefined;
 }
 
-const StyledComponentContainer = styled.div`
-  display: flex;
-  flex-direction: column;
-  gap: ${spacing.normal};
-  max-width: ${misc.maxTextWidth};
-`;
+const StyledComponentContainer = styled("div", {
+  base: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "medium",
+  },
+});
 
-const ShortInfoDiv = styled.div`
-  display: flex;
-  flex-direction: column;
-  gap: ${spacing.xxsmall};
-`;
+const ShortInfoDiv = styled("div", {
+  base: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "4xsmall",
+    maxWidth: "surface.xlarge",
+  },
+});
 
-const StyledUl = styled.ul`
-  padding-left: ${spacing.mediumlarge};
-  padding-bottom: ${spacing.small};
-`;
+const ShortInfoLine = styled("span", {
+  base: {
+    display: "flex",
+    flexDirection: "row",
+    gap: "4xsmall",
+  },
+});
+
+const StyledUl = styled("ul", {
+  base: {
+    paddingInlineStart: "xlarge",
+    paddingBlockEnd: "xxsmall",
+    listStyleType: "unset",
```

Comment:
Hvorfor trenger vi dette?

Reply:
Kulepunktet foran forsvant - denne sørget for at den dukka opp igjen.

Reply:
Som en generell tommelfingerregel: Dette er et varselflagg. Safari er blant annet vanskelige på dette: En liste som også er en flexbox telles ikke lenger som en liste. I de aller fleste tilfeller går dette fint; I dette tilfellet ville jeg sagt at dette er en veldig eksplisitt liste, som derfor også burde ha all semantikken som følger med det. Da tror jeg heller vi burde gå for en mer standard (og kjipere) løsning. `paddingBlockEnd` på alle listeelementer utenom det siste, f.eks.